From 89af648a0ab22fa6b410b5f2ad28da0f57e09199 Mon Sep 17 00:00:00 2001
From: Nico <nicorg2529@gmail.com>
Date: Mon, 7 Jan 2019 14:52:08 +0100
Subject: [PATCH 32/41] StatusBarSignalPolicy: Fix missing provisioned in
 equals and copyTo

* equals and copyTo should include all MobileIconState properties
* therefore, add the newly introduced boolean, provisioned

Change-Id: Ic810bbe7538506612f86695ce520e8b9542bc634
---
 .../android/systemui/statusbar/phone/StatusBarSignalPolicy.java | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusBarSignalPolicy.java b/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusBarSignalPolicy.java
index f4ac7e3fb2e..dfc7833082a 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusBarSignalPolicy.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusBarSignalPolicy.java
@@ -446,6 +446,7 @@ public class StatusBarSignalPolicy implements NetworkControllerImpl.SignalCallba
                     typeId == that.typeId &&
                     roaming == that.roaming &&
                     needsLeadingPadding == that.needsLeadingPadding &&
+                    provisioned == that.provisioned &&
                     Objects.equals(typeContentDescription, that.typeContentDescription);
         }
 
@@ -471,6 +472,7 @@ public class StatusBarSignalPolicy implements NetworkControllerImpl.SignalCallba
             other.typeId = typeId;
             other.roaming = roaming;
             other.needsLeadingPadding = needsLeadingPadding;
+            other.provisioned = provisioned;
             other.typeContentDescription = typeContentDescription;
         }
 
-- 
2.17.1

